src/renderer/components/properties-panel/ComputeNodesPropertiesTab.tsx

```diff
@@ -203,7 +203,11 @@ const ComputeNodePropertiesCard = ({ node, temporalEpoch, chipId }: ComputeNodeP
                                         <div style={{ fontSize: '12px' }}>
                                             <GraphVertexDetailsSelectables
                                                 operand={operand}
-                                                isOffchip={(operand as BuildableOperation)?.isOffchip}
+                                                isOffchip={
+                                                    chipId === undefined
+                                                        ? false
+                                                        : (operand as BuildableOperation)?.isOffchip
```

Comment:
this may seem like a nitpick but we should be accessing  `BuildableOperation`. is offChip was implemented on the `Operand` for a reason - its a concern separation between graph creation and graph rendering